packages/live-preview-sdk/src/inspectorMode/index.ts

```diff
@@ -19,7 +19,7 @@ type InspectorModeOptions = {
 };
 
 type TaggedElement = {
-  attributes: InspectorModeAttributes;
+  attributes: InspectorModeAttributes | null;
```

Comment:
why do we need `null` here?
Elements without attributes should not be tagged? 🤔

Reply:
`getManualInspectorModeAttributes` sets null sometimes:

>  * Parses the necessary manually tagged information from the element and returns them.
>  * If **one** of the information is missing it returns null